examples/06_joining_tables_with_FuzzyJoin.py

```diff
@@ -26,7 +26,7 @@
 import pandas as pd
 
 df = pd.read_csv(
-    "https://raw.githubusercontent.com/dirty-cat/datasets/master/data/Happiness_report_2022.csv",
+    "https://raw.githubusercontent.com/dirty-cat/datasets/main/data/Happiness_report_2022.csv",
```

Comment:
This is in the `datasets` repo of the `dirty_cat` organisation.. Keep it as master or it won't work

Reply:
Nice catch!